common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -234,6 +234,24 @@ def test_proctored_exam_flow(self):
         LogoutPage(self.browser).visit()
         self._login_as_a_verified_user()
 
+        # Immediately verify the user
```

Comment:
I wonder if you should factor this logic out into a helper, since if we add other tests they will need to do the exact same set of steps.